sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -0,0 +1,241 @@
+      adjust_campaign STRING
+    OPTIONS
+      (description = "Structure parameter for the campaign name."),
+      adjust_ad_group STRING
+    OPTIONS
+      (description = "Structure parameter for the the ad group of a campaign."),
+      adjust_creative STRING
+    OPTIONS
+      (description = "Structure parameter for the creative content of a campaign."),
+      adjust_network STRING
+    OPTIONS
+      (description = "The type of source of a client installation."),
+      install_source STRING
+    OPTIONS
+      (description = "The source of a client installation."),
+      metadata STRUCT<
+        reported_first_session_ping BOOL
+        OPTIONS
+          (description = "True if the client ever reported a first_session ping."),
+          reported_metrics_ping BOOL
+        OPTIONS
+          (description = "True if the client ever reported a metrics ping."),
+          min_first_session_ping_run_date DATE
+        OPTIONS
+          (description = "Date of first run in the earliest first_session ping reported."),
+          adjust_network__source_ping STRING
+        OPTIONS
+          (description = "Name of the ping that reported the first adjust_network value."),
+          install_source__source_ping STRING
+        OPTIONS
+          (description = "Name of the ping that reports the install_source value."),
+          adjust_network__source_ping_datetime DATETIME
+        OPTIONS
+          (description = "Datetime of the ping that reported the first adjust_network value."),
+          install_source__source_ping_datetime DATETIME
+        OPTIONS
+          (description = "Datetime of the ping that reported the first install_source value.")
+      >
+  )
+PARTITION BY
+  first_seen_date
+CLUSTER BY
+  channel,
+  sample_id,
+  first_reported_country,
+  device_model
+OPTIONS
+  (
+    description = "First observations for Firefox Android clients retrieved from the earliest pings: baseline, first_session and metrics. The attributes stored in this table include the first attribution, device, OS version and ISP. This table should be accessed through the user-facing view `fenix.firefox_android_clients`. Proposal: https://docs.google.com/document/d/12bj4DhCybelqHVgOVq8KJlzgtbbUw3f68palNrv-gaM/. For more details about attribution and campaign structure see https://help.adjust.com/en/article/tracker-urls#campaign-structure-parameters."
+  );
+
+INSERT
+  `moz-fx-data-shared-prod.fenix_derived.firefox_android_clients_v1`
+WITH first_seen AS (
+  SELECT
+    client_id,
+    sample_id,
+    first_seen_date,
+    country AS first_reported_country,
+    isp AS first_reported_isp,
+    DATETIME(first_run_date) AS first_run_datetime,
+    normalized_channel AS channel,
+    device_manufacturer,
+    device_model,
+    normalized_os_version AS os_version
+  FROM
+    `mozdata.fenix.baseline_clients_first_seen`
+  WHERE
+    submission_date >= '2021-01-01'
```

Comment:
We should include more history than this. At least until beginning of 2019.
```suggestion
    submission_date >= '2019-01-01'
```